diff --git a/dist/app/components/error-404.vue b/dist/app/components/error-404.vue
index 67d32a9a931cd174dedd1d04c62add378518fbe0..0cb90d0be2bded76e252a5bda2249ebb526d7904 100644
--- a/dist/app/components/error-404.vue
+++ b/dist/app/components/error-404.vue
@@ -23,7 +23,7 @@ const props = defineProps({
   },
   backHome: {
     type: String,
-    default: "Go back home"
+    default: "atoms was here"
   }
 })
 useHead({
