diff --git a/index.js b/index.js
index 9be86a8d6a9e00b53a23daa0c8706ddbc5a7d432..bdf10d2bafe71c8f23c480972b7fc5a05dd25b1f 100644
--- a/index.js
+++ b/index.js
@@ -2,6 +2,8 @@
 
 if (process.env.NODE_ENV === 'production') {
   module.exports = require('./dist/vue-router.prod.cjs')
+  console.log('vue-router.prod.cjs')
 } else {
   module.exports = require('./dist/vue-router.cjs')
+  console.log('vue-router.cjs')
 }
